sdk/identity/identity/src/msal/browserFlows/browserCommon.ts

```diff
@@ -53,6 +60,48 @@ export function defaultBrowserMsalConfig(
   };
 }
 
+/**
+ * Defines common properties that customizes browser authentication.
+ * The full list of available properties can be seen here: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/3a730d5ad842eaa8fc643015817cd7c20415e921/lib/msal-browser/src/request/PopupRequest.ts
+ * We will be adding more on demand.
+ */
+export interface MsalBrowserLoginOptions {
```

Comment:
If it's just about common properties to browser authentication, minus scopes, could we just use `type MsalBrowserRequestOptions = Omit<msalBrowser.PopupRequest & msalBrowser.RedirectRequest, "scopes">` (or something like that) instead of creating our own internal interface representing it?

I just think that between this and InteractiveBrowserCredentialOptions we're doing an awful lot of re-specification for something that is strongly typed already by MSAL.

Reply:
No no we don’t want to expose anything from MSAL directly.

Reply:
Well, this type is already named `MsalBrowserLoginOptions`, so it isn't really a question of exposed or not, right? It's just extraneous to specify this interface again when it's strongly typed _by MSAL itself_.

Reply:
Basically, this shape is specified three different times, why? It's specified first by MSAL itself, second here by us for our internals, and third again for our public API surface. I'm saying we can cut out the middle one and just have our public API surface one that is part of InteractiveBrowserCredentialBrowserOptions, and the fully internal one that is computed from MSAL types.

Reply:
Oh I see your point, I had the feeling this was about the public API. Alright I’ll take your feedback! Appreciated 🌞 Let me do the changes

Reply:
I’ll also add @internal to make it easier for me to see this is not public API